src/index/IndexImpl.h

```diff
@@ -334,15 +334,38 @@ class IndexImpl {
   Index::WordEntityPostings getContextEntityScoreListsForWords(
       const string& words) const;
 
-  Index::WordEntityPostings getWordPostingsForTerm(const string& term) const;
+  // Returns a superset of textRecords and their corresponding words
+  // and scores. Each textRecord has to contain the term.
+  Index::WordEntityPostings getWordPostingsForTermWep(const string& term) const;
+
+  // Returns a superset of textRecords and their corresponding words
```

Comment:
Why is this a `superset`?
Especially write what columns the IdTables have and what is contained in these columns (only the words, or maybe also other words with the same prefix etc.)

Reply:
ups. should be subset.